onie-image.conf

```diff
@@ -25,22 +25,22 @@ FILESYSTEM_DOCKERFS=dockerfs.tar.gz
 DOCKERFS_DIR=docker
 
 ## Output file name for onie installer
-OUTPUT_ONIE_IMAGE=target/sonic-$TARGET_MACHINE.bin
+OUTPUT_ONIE_IMAGE=target/${IMAGE_PREFIX}sonic-$TARGET_MACHINE.bin
 
 ## Output file name for raw image
-OUTPUT_RAW_IMAGE=target/sonic-$TARGET_MACHINE.raw
+OUTPUT_RAW_IMAGE=target/${IMAGE_PREFIX}sonic-$TARGET_MACHINE.raw
 
 ## Raw image size in MB
 RAW_IMAGE_DISK_SIZE=1024
 
 ## Output file name for kvm image
-OUTPUT_KVM_IMAGE=target/sonic-$TARGET_MACHINE.img
+OUTPUT_KVM_IMAGE=target/${IMAGE_PREFIX}sonic-$TARGET_MACHINE.img
 
 ## Kvm image size in GB
 KVM_IMAGE_DISK_SIZE=16
 
 ## Output file name for aboot installer
-OUTPUT_ABOOT_IMAGE=target/sonic-aboot-$TARGET_MACHINE.swi
+OUTPUT_ABOOT_IMAGE=target/${IMAGE_PREFIX}sonic-aboot-$TARGET_MACHINE.swi
```

Comment:
it is better to put dbg as a suffix instead of prefix. sonic-broadom-dbg.bin.

Reply:
done